packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,16 +1,46 @@
+## 45.14.0 - May 9, 2019
+
+### Added
+
+#### salesforcedx-vscode
+
+- Reference the change log at the top level of the repository ([PR #1304](https://github.com/forcedotcom/salesforcedx-vscode/pull/1304), [Issue #1292](https://github.com/forcedotcom/salesforcedx-vscode/issues/1292))
+
+### Fixed
+
+#### salesforcedx-vscode-core
+
+- When running `SFDX: Turn On Apex Debug Log for Replay Debugger`, show trace flags only for the user that Salesforce CLI is authenticated with ([PR #1315](https://github.com/forcedotcom/salesforcedx-vscode/pull/1315), [Issue #1285](https://github.com/forcedotcom/salesforcedx-vscode/issues/1285), [Issue #1280](https://github.com/forcedotcom/salesforcedx-vscode/issues/1280))
+- When running commands to create metadata, show only directories within package directories as available locations ([PR #1288](https://github.com/forcedotcom/salesforcedx-vscode/pull/1288), [Issue #1206](https://github.com/forcedotcom/salesforcedx-vscode/issues/1206))
+
+#### salesforcedx-vscode-lightning, salesforcedx-vscode-lwc
+
+- Use system version of Node.js when launching Aura Language Server and LWC Language Server ([PR #1305](https://github.com/forcedotcom/salesforcedx-vscode/pull/1305))
+- Show method icon for LWC component methods in Lightning Explorer ([PR #1305](https://github.com/forcedotcom/salesforcedx-vscode/pull/1305))
+- Display message in Lightning Explorer when no components are found ([PR #1305](https://github.com/forcedotcom/salesforcedx-vscode/pull/1305))
+- Filter Aura and LWC system namespaces for unknown workspaces in Lightning Explorer when no custom components are detected ([PR #1305](https://github.com/forcedotcom/salesforcedx-vscode/pull/1305))
```

Comment:
@lcampos - What does this line mean?

Reply:
@ruthemmanuelle I believe this has to do with how components are grouped in the lightning explorer. Custom components are under the `c` option and the rest are under their corresponding/known namespaces

<img width="301" alt="Screen Shot 2019-05-08 at 11 04 09 AM" src="https://user-images.githubusercontent.com/1041842/57397277-2bbc6e80-7181-11e9-8068-96f1174c397f.png">

@nrkruk is there a better wording for that fix ?